src/main/java/liquibase/ext/spanner/CloudSpanner.java

```diff
@@ -49,12 +57,26 @@ public boolean dataTypeIsNotModifiable(final String typeName) {
 
   @Override
   public String getDateLiteral(final String isoDate) {
-    String literal = super.getDateLiteral(isoDate);
+    String literal;
+
+    // Construct the literal based on whether it is a DATE or TIMESTAMP
     if (isDateTime(isoDate)) {
-      literal = "TIMESTAMP " + literal.replace(' ', 'T');
+      Date date;
```

Comment:
nit: move this inside the try block to what is now line 66

Reply:
I don't think this is correct, the super class already has similar behavior, and the try catch block was needed for the ISODateFormat().parse(isoDate) method.